internal/services/databricks/databricks_virtual_network_peerings_resource_test.go

```diff
@@ -0,0 +1,335 @@
+package databricks_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-azure-sdk/resource-manager/databricks/2023-02-01/vnetpeering"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+type DatabricksVirtualNetworkPeeringResource struct{}
+
+func TestAccDatabricksVirtualNetworkPeering_basic(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_databricks_virtual_network_peering", "test")
+	r := DatabricksVirtualNetworkPeeringResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep("workspace_id"),
+	})
+}
+
+func TestAccDatabricksVirtualNetworkPeering_requiresImport(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_databricks_virtual_network_peering", "test")
+	r := DatabricksVirtualNetworkPeeringResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.RequiresImportErrorStep(r.requiresImport),
+	})
+}
+
+func TestAccDatabricksVirtualNetworkPeering_update(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_databricks_virtual_network_peering", "test")
+	r := DatabricksVirtualNetworkPeeringResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.basic(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep("workspace_id"),
+		{
+			Config: r.update(data),
+			Check: acceptance.ComposeTestCheckFunc(
+				check.That(data.ResourceName).ExistsInAzure(r),
+			),
+		},
+		data.ImportStep("workspace_id"),
```

Comment:
`workspace_id` is being set in the Read function - why is this needed?

```suggestion
		data.ImportStep(),
```

Reply:
Same as above.